### DISCUSSION TITLE
Why freeplane can't show this picture in note panel? (resolved)

### DISCUSSION THREAD

**Discussion by guoranjun2** [freeplane/freeplane#871] [Help] [answered]
**I'm so sorry, it turn to be my network error.**

> ![](https://qph.cf2.quoracdn.net/main-qimg-d58eec4ff712b29b27be5558dca06b66-lq)
> 
> you can put this code below in freeplane content dialog for test.
> ```html
> <img src="https://qph.cf2.quoracdn.net/main-qimg-d58eec4ff712b29b27be5558dca06b66-lq">
> ```
> ![image](https://user-images.githubusercontent.com/50692082/206430463-8ba61a3f-0ab1-42cb-949a-cbde6da53f01.png)
> 
> and even you download it to your local directory and switch link, it still can't work, is it bug?

**Comment by sgjohnson1981** [comment]
Rather than marking a title "resolved," it appears the preferred method is to mark a reply as an "answer" using the Mark As Answer button. And you can mark your own replies as answers.

**Reply by guoranjun2** [reply]
ok. thanks for pointing out that.

**Comment by guoranjun2** [comment, marked as answer]
it turn to be my network error.
